ert/storage/_storage.py

```diff
@@ -85,17 +85,18 @@ async def _transmit_recordtree(
         storage_url = self._uri[: self._uri.rfind("/")]
         for record_path in record.flat_record_dict:
             record_name = record_path.split("/")[-1]
-            transmitter = StorageRecordTransmitter(record_name, storage_url)
+            transmitter = StorageRecordTransmitter(
+                record_name, storage_url, iens=self._real_id
```

Comment:
`iens` is a mnemonic used for legacy reasons I guess. Probably another PR to change that to something more precise, like "`real_id`"

Reply:
Fully agree there...